iote/bricks-angular/src/lib/material-form-bricks/components/autocomplete-action-field/autocomplete-action-field.component.ts

```diff
@@ -17,6 +17,7 @@ export class AutocompleteActionFieldComponent<T> implements OnInit, OnChanges
   @Input() required: boolean;
   @Input() icon :string;
   @Input() inputType :string = 'text';
+  @Input() isDisableSupplier : boolean;
```

Comment:
@noellamwanzia  (Minor change) Rename the variable here. Since this is a reusable component shared across multiple apps. It's not intended to only be used in selection of Suppliers alone.